src/components/ADempiere/Form/VPOS/Collection/convertAmount/index.vue

```diff
@@ -21,7 +21,12 @@
       <span>
         <b>
           {{ $t('form.pos.collect.convertAmount') }}:
-          {{ formatPrice(amountConvertionTotal, displayCurrency) }}
+          {{
```

Comment:
Just can see change for soiurce code with the same functionality, please let it without changes